sale_by_packaging/models/product_packaging.py

```diff
@@ -18,3 +19,8 @@ class ProductPackaging(models.Model):
         "When the user will put 3 as quantity, the system can force the "
         "quantity to the superior unit (5 for this example).",
     )
+
+    @api.depends("packaging_type_id")
```

Comment:
```suggestion
    @api.depends("packaging_type_id.can_be_sold")
```

Reply:
@rousseldenis I think this is done on purpose:  you don't want this value to change if the type changes its value.

Reply:
Yes that is how it was requested.
You get the default "can_be_sold" value when selecting a packaging_type. But afterwards you can change it on the product.packaging.

Reply:
@rousseldenis ping :)